### PR TITLE
Marc/edge28 plugin scaling

### PR DESCRIPTION
**Pull request by NubeDev**

### CODE REVIEW

File: database/point.go [hunk 1]
```diff
@@ -290,31 +290,16 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
...
-	body.OriginalValue = &value
 	var limitMin *float64
 	if body.LimitMin == nil {
 		if pointModel.LimitMin != nil {
 			limitMin = pointModel.LimitMin
 		} else {
 			limitMin = body.LimitMin
 		}
+	} else {
+		limitMin = body.LimitMin
```
Comment (RaiBnod) [inline database/point.go:302]: On its if condition, just remove those if/else case that's useless.

File: database/point.go [hunk 2]
```diff
@@ -323,6 +308,8 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 		} else {
 			limitMax = body.LimitMax
 		}
+	} else {
+		limitMax = body.LimitMax
```
Comment (RaiBnod) [inline database/point.go:312]: Same as in above.

File: database/point.go [hunk 3]
```diff
@@ -331,6 +318,8 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 		} else {
 			scaleInMin = body.ScaleInMin
 		}
+	} else {
+		scaleInMin = body.ScaleInMin
```
Comment (RaiBnod) [inline database/point.go:322]: Same as in above.

File: database/point.go [hunk 4]
```diff
@@ -339,6 +328,8 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 		} else {
 			scaleInMax = body.ScaleInMax
 		}
+	} else {
+		scaleInMax = body.ScaleInMax
```
Comment (RaiBnod) [inline database/point.go:332]: Same as in above.

File: database/point.go [hunk 5]
```diff
@@ -347,6 +338,8 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 		} else {
 			scaleOutMin = body.ScaleOutMin
 		}
+	} else {
+		scaleOutMin = body.ScaleOutMin
```
Comment (RaiBnod) [inline database/point.go:342]: Same as in above.

File: database/point.go [hunk 6]
```diff
@@ -355,6 +348,8 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 		} else {
 			scaleOutMax = body.ScaleOutMax
 		}
+	} else {
+		scaleOutMax = body.ScaleOutMax
```
Comment (RaiBnod) [inline database/point.go:352]: Same as in above.

File: database/point.go [hunk 7]
```diff
@@ -368,6 +363,7 @@ func (d *GormDatabase) UpdatePointValue(uuid string, body *model.Point, fromPlug
 				if val == nil {
 					priority[typeOfPriority.Field(i).Name] = nil
 				} else {
+					fmt.Println("UpdatePointValue() - i, val:", typeOfPriority.Field(i).Name, *val)
```
Comment (RaiBnod) [inline database/point.go:366]: Use logger:

```go
import log "github.com/sirupsen/logrus"
log.Info("<msg>")
```

File: plugin/nube/protocals/edge28/README.md [hunk 8]
```diff
@@ -21,7 +21,7 @@ write to DOs or R1, R2 can write true/false/1/0
 
 ``` 
 http://192.168.15.101:5000/api/1.1/write/do/r1/false/16
-http://192.168.15.101:5000/api/1.1/write/do/r1/1/16
+http://192.168.15.101:5000`/api/1.1/write/do/r1/1/16`
```
Comment (RaiBnod) [inline plugin/nube/protocals/edge28/README.md:24]: UPdate this to: `http://192.168.15.101:5000/api/1.1/write/do/r1/1/16`

File: utils/bool.go [hunk 9]
```diff
@@ -25,3 +25,11 @@ func IsTrue(b *bool) bool {
...
 	}
 }
+
+func BoolIsNilCheck(b *bool) bool {
+	if b == nil {
+		return true
+	} else {
+		return false
+	}
+}
```
Comment (RaiBnod) [inline utils/bool.go:35]: Remove this one, this function isn't in use and if needed we can directly do `val == nil`.

**Review by RaiBnod — CHANGES_REQUESTED** [review]
Please have those changes.

**Comment by RaiBnod** [comment]
And also check why that CI is failed.

**Comment by CanuckMarc** [comment]
@RaiBnod Please check the build error, fix, and merge.

**Pull request merged**